src/types/ui/Multiselect.ts

```diff
@@ -10,6 +10,6 @@ export interface IServiceOptions {
 export interface MultiSelectProps {
```

Comment:
The letter "I" is missing, which refers to "Interface".

Reply:
Done